Resources/Prototypes/Stacks/floor_tile_stacks.yml

```
@@ -203,70 +203,80 @@
 
 - type: stack
   id: FloorCarpetRed
-  name: red carpet tile
+  name: red carpet
+  icon: { sprite: Objects/Tiles/tile.rsi, state: carpet-red }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)